transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_self_hosted_servers.sql

```diff
@@ -0,0 +1,21 @@
+with mm_telemetry_prod_license as (
+    select timestamp::date as license_date
+        , license_id as license_id
+        , server_id as server_id
+        , customer_id as customer_id
+        , license_name as license_name
+    from {{ ref('stg_mm_telemetry_prod__license')}} 
+    where license_id is not null and installation_id is null
+    qualify row_number() over (partition by server_id order by timestamp desc) = 1
+), mattermost2_license as (
+    select timestamp::date as license_date
+        , license_id as license_id
+        , server_id as server_id
+        , customer_id as customer_id
+        , license_name as license_name
+    from {{ ref('stg_mattermost2__license')}} 
+    where license_id is not null
+    qualify row_number() over (partition by server_id order by timestamp desc) = 1
+) select * from mm_telemetry_prod_license
```

Comment:
Servers in a few versions sent telemetry both to segment and rudderstack. Using `union` will most likely add two rows for each server.

Reply:
I checked the documentation and the data. `union all` adds the duplicate rows and `union` only keeps unique records.

Reply:
We are good here.